src/diracx/db/job_parameters/db.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from opensearchpy import OpenSearch
+
+try:
+    from opensearchpy import A, Q, Search
+except ImportError:
+    pass
+
+from ..utils import BaseDB
+
+
+class OpenSearchJobParametersDB(BaseDB):
+    async def set_client():
+        # TODO: get connection secrets from SettingsClass.create and connection details from configuration
+        host = "opensearch-cluster-master"
+        port = 9200
+        # For now credentials are hard coded, to be set here
+        auth = ("user", "pwd")
```

Comment:
Note that these are fake credentials. In order to make it work in a demo environment, one should substitute them with the correct ones.